### PR TITLE
KATI-95_Request-List-Function

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java [hunk 1]
```diff
@@ -46,8 +52,35 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
             public void onRefresh() {
-                Toast.makeText(getActivity(),"Refreshing",Toast.LENGTH_LONG).show();
-                mSwipeRefreshLayout.setRefreshing(false);
+                AsyncTask<Object, Void, Void> task = new AsyncTask<Object, Void, Void>() {
+                    private Exception exception = null;
+
+                    @Override
+                    protected Void doInBackground(Object[] params) {
+                        try {
+                            DoctorUserEntity doctor = (DoctorUserEntity) CurrentUserProfile.getInstance().getEntity();
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java:61]: Again, do not use CurrentUserProfile in Model :-1:

Comment (vtnlinh95) [inline app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java:61]: @hieumai it is not in Model, it is in asyntask

File: app/src/main/java/com/kms/cura/model/AppointmentModel.java [hunk 2]
```diff
@@ -47,7 +48,13 @@ public static AppointmentModel getInstance() {
         StringBuilder builder = new StringBuilder();
         builder.append(Settings.SERVER_URL);
         builder.append(Settings.GET_APPT);
-        AppointmentModelResponse response = new AppointmentModelResponse(search.getAppointmentEntity().getPatientUserEntity());
+        AppointmentModelResponse response = null;
+        if (CurrentUserProfile.getInstance().isPatient()) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/model/AppointmentModel.java]: Here, i mean do not use CurrentUserProfile here :8ball:

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by hieumai** [comment]
@vtnlinh95  please rebase this

**Comment by vtnlinh95** [comment]
@hieumai rebased

**Pull request merged**